rest-api-with-fastify/spec/romcal.spec.js

```diff
@@ -0,0 +1,53 @@
+import * as chai from 'chai';
+import chaiHttp from 'chai-http';
+import fastify from '../app.js';
+
+const chaiModule = chai.use(chaiHttp);
+const { request } = chaiModule;
+
+chai.use(chaiHttp);
+const { expect } = chai;
+
+let server;
+
+before(async () => {
+  const init = await fastify.listen({ port: 3000 });
```

Comment:
I wish we could change the port using an env var (e.g. `ROMCAL_APP_PORT` or whatever env var name you prefer).

Reply:
asbefore, will do